src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java14/InputJava14InstanceofWithPatternMatching.java

```diff
@@ -59,18 +63,154 @@ public void setModel(String model) {
+            System.out.println("Blah");
+        }
+
+        if (!(o1 instanceof String k)) {
+            return ;
+        }
+
+        if (o1 instanceof String s2) {}
+
+
+        if (!(o1 instanceof String s4)) {
+            return ;
+        }
+
+        boolean result = (o1 instanceof String a1) ?
+                (o1 instanceof String a2) : (!(o1 instanceof String a3));
+
+        if (!(o1 instanceof String s) ? false : s.length()>0){
+            System.out.println("done");
+        }
+
+        if ((o1 instanceof String s) ? s.length()>0 : false){
+            System.out.println("done");
+        }
+
+        if (!(o1 instanceof String s) || !(o2 instanceof Integer i)){
+        } else {
+            s.length();
+            i.intValue();
+        }
+
+        if (!(o1 instanceof String s) || !(o2 instanceof Integer i)){
+        } else {
+            s.length();
+            i.intValue();
+        }
+
+        if (o1 instanceof String s && o2 instanceof Integer in) {
+            s.length();
+            in.intValue();
+        }
+
+        L1: {
+            if (o1 instanceof String s) {
+                s.length();
+            } else {
+                break L1;
+            }
+            s.length();
+        }
+
+        {
+            L2: for ( ;!(o1 instanceof String s); ) {
+
+            }
+
+            s.length();
+        }
+
+        {
+            while (!(o1 instanceof String s)) {
+                L3: break L3;
+            }
+
+            while(o1 instanceof String str) {
+                str.length();
+            }
+        }
+
+        if (!new VoidPredicate() { public boolean get() { return o1 instanceof String str
+                && !str.isEmpty();} }.get()) {
+            throw new AssertionError();
+        }
+
+        if (!((VoidPredicate) () -> o1 instanceof String str && !str.isEmpty()).get()) {
+            throw new AssertionError();
+        }
+
+        if (o1 instanceof String j && j.length() == 5 && o2 instanceof Integer z && z == 42) {
+            System.out.println(j);
+            System.out.println(z);
+        } else {
+        }
+
+        int x = o1 instanceof String j ? j.length() : 2;
+
+        x = !(o1 instanceof String j) ? 2 : j.length();
+
+        Object ch = null;
+
+        for (; o1 instanceof String j; j.length()) {
+            System.out.println(j);
+        }
+
+        String formatted;
+        if (o1 instanceof Integer i) formatted = String.format("int %d", i);
+        else if (o1 instanceof Byte by) formatted = String.format("byte %d", by);
+        else if (o1 instanceof Long l) formatted = String.format("long %d", l);
+        else if (o1 instanceof Double d) formatted = String.format("double %f", d);
+        else if (o1 instanceof String s) formatted = String.format("String %s", s);
+        else formatted = String.format("Something else "+ o1.toString());
+
+        do {
+            L4: break L4;
+        } while (!(o1 instanceof String s));
+    }
```

Comment:
Please add a code snippet with the full class name, something like
```java
if (o instanceof org.example.Foo foo)
```

Reply:
Fixed.